monitor/linuxmonitor/linuxMonitor.go

```diff
@@ -18,25 +18,48 @@ import (
 	"github.com/shirou/gopsutil/process"
 )
 
-// SystemdRPCMetadataExtractor is a systemd based metadata extractor
-func SystemdRPCMetadataExtractor(event *rpcmonitor.EventInfo) (*policy.PURuntime, error) {
+// DefaultHostMetadataExtractor is a host specific metadata extractor
+func DefaultHostMetadataExtractor(event *rpcmonitor.EventInfo) (*policy.PURuntime, error) {
 
-	if event.Name == "" {
-		return nil, fmt.Errorf("EventInfo PU Name is empty")
+	runtimeTags := policy.NewTagStore()
+
+	for _, tag := range event.Tags {
+		parts := strings.Split(tag, "=")
+		if len(parts) != 2 {
+			return nil, fmt.Errorf("Invalid Tag")
+		}
+		runtimeTags.AppendKeyValue("@usr:"+parts[0], parts[1])
 	}
 
-	if event.PID == "" {
-		return nil, fmt.Errorf("EventInfo PID is empty")
+	options := &policy.OptionsType{
+		CgroupName: event.PUID,
+		CgroupMark: strconv.FormatUint(cgnetcls.MarkVal(), 10),
+		Services:   event.Services,
 	}
 
-	if event.PUID == "" {
-		return nil, fmt.Errorf("EventInfo PUID is empty")
+	runtimeIps := policy.ExtendedMap{"bridge": "0.0.0.0/0"}
```

Comment:
this appears several times. You should have a `policy.NewExtendedMap(map[string]string)` so it can be initialized from a single place.

Reply:
This is another refactor that needs to happen that can use the new sync maps.